DevProxy.Plugins/Utils/GraphUtils.cs

```diff
@@ -96,4 +101,57 @@ internal async Task<IEnumerable<string>> UpdateUserScopesAsync(IEnumerable<strin
 
         return newMinimalScopes;
     }
+
+    public static MethodAndUrl GetMethodAndUrl(string methodAndUrlString)

```

Comment:
This method doesn't need to be public. Let's make it internal instead

Reply:
fixed